model/registry/go.yaml

```
@@ -0,0 +1,25 @@
+groups:
+  - id: registry.go
+    type: attribute_group
+    prefix: go
+    brief: >
+      This document defines Go related attributes.
+    attributes:
+      - id: memory.type
+        stability: experimental
+        type:
+          allow_custom_values: false
+          members:
+            - id: stack
+              value: 'stack'
+              brief: 'Memory allocated from the heap that is reserved for stack space, whether or not it is currently in-use.'
+              note: >
+                Computed from `/memory/classes/heap/stacks:bytes`.
+              stability: experimental
+            - id: other
+              value: 'other'
+              brief: 'Memory used by the Go runtime, excluding other categories of memory usage.'
```

I don't understand the meaning of this phrase - what is `other categories of memory` ? I see here only stack above and that's all?

---

Thats correct. It currently includes all memory except `stack`.  I phrased it that way to make it possible to extend the metric with future values.  If other categories are introduced (e.g. `heap`), the `other` category would then be all memory except `heap` and `stack`. It exists to ensure the total of all categories is the total memory usage.

---

```suggestion
              brief: 'Memory used by the Go runtime, excluding other categories of memory usage described in this enumeration.'
```

what about this option?

---

sure, thats fine.